src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1050,23 +1169,6 @@ public bool IsTopMostNode
             }
         }
 
-        internal void ConnectInput(int inputData, int outputData, NodeModel node)
```

Comment:
@mjkkirschner is correct. For large graphs, freeze was very slow. so, we cached these values. this is when you load a large graph with freeze nodes.